src/pages/Profile/index.js

```diff
@@ -53,39 +64,49 @@ function Profile() {
     });
   };
   return (
-    <div className="profile-container">
-      <div className="profile">
-        <div>
-          <img className="profile-img" src={userData.profile_picture} />
-        </div>
-        <div className="profile-details">
+    <Container maxWidth="lg" sx={{ pt: 20 }}>
+      <Grid container spacing={2}>
+        <Grid item xs={12} md={6} align={isActive ? "center" : "right"}>
+          <Avatar
+            alt="Remy Sharp"
+            src={userData.profile_picture}
+            sx={{ width: 100, height: 100 }}
+          />
+        </Grid>
+        <Grid item xs={12} md={6} align={isActive ? "center" : "none"}>
```

Comment:
Similar comments to above.  Remember also that `none` is not a valid value for `align`:
```suggestion
        <LeftCenterAlignedGrid item xs={12} sm={6}>
```